src/components/time-picker/time-picker.st.css

```diff
@@ -0,0 +1,101 @@
+@namespace 'time-picker';
+
+:import {
+    -st-from: "../stepper/stepper.st.css";
+    -st-default: Stepper;
+}
+
+:import {
+    -st-from: "../../style/default-theme/variables.st.css";
+    -st-named:
+        BK2,
+        BK3,
+        PR2,
+        PR3;
+}
+:vars {
+    height: 36px;
+    width: 120px;
+}
+.root {
+    -st-states: focus, disabled;
+    font-family: arial;
+    font-size: 16px;
+    position: relative;
+    border: 1px solid value(BK3);
+    height: value(height);
+    width: value(width);
```

Comment:
what will happen if user will change font only in that component, without changing constraints, it will break the view